client/py/examples/basic_read_write.py

```diff
@@ -7,40 +7,35 @@
 #  License, use of this software will be governed by the Apache License, Version 2.0,
 #  included in the file licenses/APL.txt.
 
-
 """
 This example demonstrates the basics of reading and writing data from an index and data
-channel in Synnax. We'll write a linearly increasing line of data to a data channel and
-read it back to plot it.
+channel in Synnax. We'll write data to an index and data channel in the Synnax cluster,
+and then read the data back from the cluster and plot it.
 """
 
 import matplotlib.pyplot as plt
 import numpy as np
-
 import synnax as sy
 
-
-# We've logged in via the CLI, so there's no need to provide credentials here.
-# See https://docs.synnaxlabs.com/reference/python-client/get-started for more information.
+# We've logged in via the command-line interface, so there's no need to provide
+# credentials here. See https://docs.synnaxlabs.com/reference/python-client/get-started.
 client = sy.Synnax()
 
 # Create an index channel that will be used to store our timestamps.
 time_channel = client.channels.create(
-    name="basic_read_write_time",
-    is_index=True,
-    data_type=sy.DataType.TIMESTAMP,
+    name="basic_read_write.time", is_index=True, data_type=sy.DataType.TIMESTAMP
```

Comment:
I don't think we should be recommending using periods in channel names, as it messes with things like accessing channels are property field on ranges.